app/components/form/ActiveRequestBox.tsx

```diff
@@ -10,23 +10,40 @@ import {
   IconPoint,
   IconBan,
 } from '@tabler/icons-react';
+import { useQuery } from '@tanstack/react-query';
 import { useRouter } from 'next/navigation';
+import { useSession } from 'next-auth/react';
+import { getCommentCount } from '@/services/backend/private-cloud/products';
 
 export default function ActiveRequestBox({
   data,
   className,
+  showCount = false,
 }: {
   data?: {
     cloud: 'private-cloud' | 'public-cloud';
     id: string;
+    licencePlate: string;
     type: $Enums.RequestType;
     active: boolean;
     decisionStatus: $Enums.DecisionStatus;
     createdByEmail: string;
   };
   className?: string;
+  showCount?: boolean;
 }) {
   const router = useRouter();
+  const { data: session } = useSession();
+
+  const canViewComments = session?.permissions?.viewAllPrivateProductComments;
+
+  const { data: commentData, isLoading } = useQuery({
+    queryKey: ['commentCount', data?.id],
+    queryFn: () => getCommentCount(data?.licencePlate as string, data?.id as string),
+    enabled: showCount && canViewComments && !!data?.id && !!data?.licencePlate,
```

Comment:
Let's encapsulate the entire statement as an argument so that the variable can be reused on line 142.